app/services/paper_converter.rb

```diff
@@ -21,4 +19,14 @@ def self.connection
       config.adapter :net_http
     end
   end
+
+  def self.post_ihat_job(file)
+    connection.post("/jobs", job: {
+      input: file,
+      options: {
+        recipe_name: "html_to_docx"
+      }
+    }).body
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.